modules/common/src/test/scala/com/snowplowanalytics/snowplow/storage/bigquery/common/SpecHelpers.scala

```diff
@@ -579,7 +579,7 @@ object SpecHelpers {
     private val retrySettings   = BigQueryRetrySettings(initialDelay, delayMultiplier, maxDelay, totalTimeout)
 
     private val loader: Config.Loader =
-      Config.Loader(lInput, lOutput, loadMode, consumerSettings, sinkSettings, retrySettings)
+      Config.Loader(lInput, lOutput, loadMode, consumerSettings, sinkSettings, retrySettings, FiniteDuration(60, SECONDS))
```

Comment:
Do you mind if I make the timeout duration a separate `private val`, just to be consistent with the rest of the file?